hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -25,5 +24,852 @@
       stage: V1
       tests: ["usbdev_smoke"]
     }
+    {
+      name: rst_ni
```

Comment:
This is already covered by
[auto_csr_tests](https://opentitan.org/book/hw/dv/sv/csr_utils/index.html?highlight=csr_hw_reset#csr-sequence-library)
Please remove it